tools/exports.js

```diff
@@ -17,3 +17,4 @@ exports["string_template"] = string_template;
 exports["tokenizer"] = tokenizer;
 exports["is_identifier"] = is_identifier;
 exports["SymbolDef"] = SymbolDef;
+exports["EXPECT_DIRECTIVE"] = EXPECT_DIRECTIVE;
```

Comment:
I don't think DEBUG is needed in this case. If something similar comes up a few more times then it can be revisited.